hellabot.go

```diff
@@ -356,8 +350,46 @@ var pingPong = &Trigger{
 	func(m *Message) bool {
 		return m.Command == "PING"
 	},
-	func(irc *IrcCon, m *Message) bool {
-		irc.Send("PONG :" + m.Content)
+	func(bot *Bot, m *Message) bool {
+		bot.Send("PONG :" + m.Content)
 		return true
 	},
 }
+
+type Message struct {
+	// irc.Message from sorcix
+	irc.Message
```

Comment:
could probably make `Message` embed a pointer to `irc.Message` so we dont have to do an ugly dereference down below.